clients/rust/tests/validator_stake_tokens.rs

```diff
@@ -40,12 +41,20 @@ async fn stake_tokens() {
     );
     let mut context = program_test.start_with_context().await;
 
-    // Given a config and stake accounts.
+    // Given a config account and a validator stake with 50 SOL staked.
 
     let config_manager = ConfigManager::new(&mut context).await;
     let stake_manager = ValidatorStakeManager::new(&mut context, &config_manager.config).await;
 
-    // And we initialize the holder rewards accounts and mint 100 tokens.
+    let mut account = get_account!(context, stake_manager.stake);
+    let mut stake_account = ValidatorStake::from_bytes(account.data.as_ref()).unwrap();
+    // "manually" set the amount to 50 SOL
+    stake_account.total_staked_lamports_amount = 50 * 1_000_000_000;
```

Comment:
nit: How about setting this or the staked PAL amount to exactly the limit allowable for the validator? It should be 77 lamports if I got the rounding correct?

Reply:
Good point!